src/plugin/scoring-metadata/src/java/org/apache/nutch/scoring/metadata/package.html

```diff
@@ -0,0 +1,10 @@
+<html>
+  <body>
+    <p>
+      Metadata Scoring Plugin
+    </p>
+    <p>
+      Propagates Meta data, injected from in db, parse data or content from one stage to another. 
```

Comment:
Could be more detailed, cf. [urlmeta package description](/apache/nutch/blob/master/src/plugin/urlmeta/src/java/org/apache/nutch/scoring/urlmeta/package.html), at least the 3 steps (resp. 4 containers) should be in the right order: CrawlDb --> content --> parse data --> outlinks